docs/framework/wcf/feature-details/how-to-create-temporary-certificates-for-use-during-development.md

```diff
@@ -7,50 +7,44 @@ helpviewer_keywords:
 >   
->  By default, the [Makecert.exe (Certificate Creation Tool)](http://msdn.microsoft.com/library/b0343f8e-9c41-4852-a85c-f8a0c408cf0d) creates certificates whose root authority is called "Root Agency**."** Because the "Root Agency" is not in the Trusted Root Certification Authorities store, this makes these certificates insecure. Creating a self-signed certificate that is placed in the Trusted Root Certification Authorities store enables you to create a development environment that more closely simulates your deployment environment.  
+>  By default, the [New-SelfSignedCertificate](https://docs.microsoft.com/en-us/powershell/module/pkiclient/new-selfsignedcertificate?view=win10-ps) creates certificates whose root authority is called "Root Agency**."** Because the "Root Agency" is not in the Trusted Root Certification Authorities store, this makes these certificates insecure. Creating a self-signed certificate that is placed in the Trusted Root Certification Authorities store enables you to create a development environment that more closely simulates your deployment environment.  
   
  For more information about creating and using certificates, see [Working with Certificates](../../../../docs/framework/wcf/feature-details/working-with-certificates.md). For more information about using a certificate as a credential, see [Securing Services and Clients](../../../../docs/framework/wcf/feature-details/securing-services-and-clients.md). For a tutorial about using Microsoft Authenticode technology, see [Authenticode Overviews and Tutorials](http://go.microsoft.com/fwlink/?LinkId=88919).  
   
 ### To create a self-signed root authority certificate and export the private key  
   
-1.  Use the MakeCert.exe tool with the following switches:  
-  
-    1.  `-n` `subjectName`. Specifies the subject name. The convention is to prefix the subject name with "CN = " for "Common Name".  
-  
-    2.  `-r`. Specifies that the certificate will be self-signed.  
-  
-    3.  `-sv` `privateKeyFile`. Specifies the file that contains the private key container.  
-  
-     For example, the following command creates a self-signed certificate with a subject name of "CN=TempCA."  
-  
-    ```  
-    makecert -n "CN=TempCA" -r -sv TempCA.pvk TempCA.cer  
-    ```  
-  
-     You will be prompted to provide a password to protect the private key. This password is required when creating a certificate signed by this root certificate.  
-  
+The following command creates a self-signed certificate with a subject name of "RootCA" in the user MY store. 
+```
+PS $rootCert = New-SelfSignedCertificate -CertStoreLocation cert:\CurrentUser\My -DnsName "RootCA" -TextExtension @("1.3.6.1.4.1.311.21.10={text}1.3.6.1.5.5.7.3.1,1.3.6.1.5.5.7.3.2")
+```
+Then, we convert the password to a secure string and store the path to the certificate in a variable "rootCertPath".
+```
```

Comment:
I would change this to:  
We need to export the certificate to a PFX file so that it can be imported to where it's needed in a later step. When exporting a certificate with the private key, a password is needed to protect it. We save the password in a `SecureString` and use the Export-PfxCertificate cmdlet to export the certificate with the associated private key to a PFX file. We also save just the public certificate into a .crt file using the Export-Certificate cmdlet.  

Then put all four commands in a single block.